src/frontend/apps/desk/src/features/mail-domains/components/MailDomainsContent.tsx

```diff
@@ -69,37 +84,49 @@ export function MailDomainsContent({ mailDomain }: { mailDomain: MailDomain }) {
     mailDomain && data?.results?.length
       ? data.results.map((mailbox: MailDomainMailbox) => ({
           email: `${mailbox.local_part}@${mailDomain.name}`,
+          name: `${mailbox.first_name} ${mailbox.last_name}`,
           id: mailbox.id,
         }))
       : [];
 
   useEffect(() => {
     setPagesCount(data?.count ? Math.ceil(data.count / pageSize) : 0);
   }, [data?.count, pageSize, setPagesCount]);
+
   return isLoading ? (
     <Box $align="center" $justify="center" $height="100%">
       <Loader />
     </Box>
   ) : (
     <>
+      <GlobalStyle />
+
       {isCreateMailboxFormVisible && mailDomain ? (
         <CreateMailboxForm
           mailDomain={mailDomain}
           closeModal={() => setIsCreateMailboxFormVisible(false)}
         />
       ) : null}
+
       <TopBanner
         name={mailDomain.name}
         setIsFormVisible={setIsCreateMailboxFormVisible}
+        abilities={mailDomain.abilities}
       />
+
       <Card
         $padding={{ bottom: 'small' }}
         $margin={{ all: 'big', top: 'none' }}
         $overflow="auto"
       >
         {error && <TextErrors causes={error.cause} />}
+
         <DataGrid
           columns={[
+            {
+              field: 'name',
+              headerName: t('Names'),
```

Comment:
I would rather do like that instead of the `GlobalStyle`, it is more clear I find, wdyt ?

```suggestion
              field: 'name',
              headerName: t('Names'),
              renderCell: ({ row }) => (
                <Text
                  $weight="bold"
                  $theme="primary"
                  $css="text-transform: capitalize;"
                >
                  {row.name}
                </Text>
              ),
```